src/integrations/prefect-github/prefect_github/repository.py

```diff
@@ -121,7 +121,8 @@ async def get_directory(
         # Clone to a temporary directory and move the subdirectory over
         with TemporaryDirectory(suffix="prefect") as tmp_dir:
             tmp_path_str = tmp_dir
-            cmd += f" {tmp_path_str}"
+            # wrap the directory with quotes, because shlex removes windows-style slashes "//" - fixes issue 13180 
```

Comment:
strip whitespace
```suggestion
            # wrap the directory with quotes, because shlex removes windows-style slashes "//" - fixes issue 13180
```